addons/mail/tests/test_res_partner.py

```diff
@@ -121,6 +121,21 @@ def test_discuss_mention_suggestions_priority(self):
         for i in range(0, 2):
             mail_new_test_user(self.env, login=f'{name}-{i}-portal-user', groups='base.group_portal')
             mail_new_test_user(self.env, login=f'{name}-{i}-internal-user', groups='base.group_user')
+
+        # suggest portal user of this company in another company
+        suggested_partners = (
+            self.env["res.partner"]
+            .with_user(self.company_2.user_ids[0])
+            .with_company(self.company_2)
+            .get_mention_suggestions("portal-user")
+        )
```

Comment:
That is quite random and non explicit ... use user_employee_c2. Also avoid multi-line writing like that, it takes much place and is hard to read for no reason.